src/System.Security.Cryptography.Cng/tests/AesCngTests.cs

```diff
@@ -90,5 +89,10 @@ public static bool SupportsPersistedSymmetricKeys
         {
             get { return SymmetricCngTestHelpers.SupportsPersistedSymmetricKeys; }
         }
+
+        public static bool IsAdministrator
+        {
+            get { return SymmetricCngTestHelpers.IsAdministrator; }
+        }
```

Comment:
I know you haven't been a fan of the syntax in the past, but wouldn't this look so nice and concise as:
```C#
public static bool IsAdministrator => SymmetricCndTestHelpers.IsAdministrator;
```
? :smile:  What you have is of course fine.